LibationWinForms/BookLiberation/DecryptForm.cs

```diff
@@ -56,15 +56,22 @@ public void SetNarratorNames(string narratorNames)
         private void updateBookInfo()
             => bookInfoLbl.UIThread(() => bookInfoLbl.Text = $"{title}\r\nBy {authorNames}\r\nNarrated by {narratorNames}");
 
-        public void SetCoverImage(byte[] coverBytes)
-            => pictureBox1.UIThread(() => pictureBox1.Image = ImageReader.ToImage(coverBytes));
+        public void SetCoverImage(System.Drawing.Image coverImage)
```

Comment:
May you never know the embarrassment of having someone else correct younger-you's slop/sloth

Reply:
Oh come now, that's a very minor sin!

Reply:
A thousand cuts. You've done nothing wrong; this is exactly how it should go. It's been a humbling experience though. Somehow work code and hobby code feel different; more vulnerable. It's been a growing experience for me.